gap_analysis_quotation/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_quotation}
+==============
+
+having cost estimation and a new wizard to generate proposal/quotation based on the cost/price estimation on the business requirements.
```

Comment:
This module is part of a set ("Gap Analysis") and extends the functionality
adding the possibility to generate/update a sales quotation based on the 
cost/price estimation of the business requirements.